implementation/test/TBTCSystemTest.js

```diff
@@ -9,30 +9,41 @@ const {BN, expectRevert} = require("@openzeppelin/test-helpers")
 const {expect} = require("chai")
 
 const TBTCSystem = contract.fromArtifact("TBTCSystem")
+const TestTBTCDepositToken = contract.fromArtifact("TestTBTCDepositToken")
 
 describe("TBTCSystem", async function() {
   let tbtcSystem
   let ecdsaKeepFactory
+  let tdt
 
   before(async () => {
-    const {tbtcSystemStub, ecdsaKeepFactoryStub} = await deployAndLinkAll(
+    const {
+      tbtcSystemStub,
+      ecdsaKeepFactoryStub,
+      tbtcDepositToken,
+    } = await deployAndLinkAll(
       [],
       // Though deployTestDeposit deploys a TBTCSystemStub for us, we want to
       // test TBTCSystem itself.
-      {TBTCSystemStub: TBTCSystem},
+      {TBTCSystemStub: TBTCSystem, tbtcDepositToken: TestTBTCDepositToken},
```

Comment:
Do we need to add `tbtcDepositToken: TestTBTCDepositToken` here? It's already initialized in the helper function:
https://github.com/keep-network/tbtc/blob/83559cb26805b5c94106e0a3a99a68ca5c4a4428/implementation/testHelpers/testDeployer.js#L23

Reply:
The utility will replace the `tbtcDepositToken` used to initialize the `Deposit` and all related contracts. So `tbtcDepositToken` refers to `TestTBTCDepositToken`. We need this because the `tbtcDepositToken` doens't have a `forceMint`

Reply:
> The utility will replace the tbtcDepositToken used to initialize the Deposit and all related contracts. So tbtcDepositToken refers to TestTBTCDepositToken.

But I believe we don't need to replace it as it's already configured to `estTBTCDepositToken` [there](https://github.com/keep-network/tbtc/blob/83559cb26805b5c94106e0a3a99a68ca5c4a4428/implementation/testHelpers/testDeployer.js#L23), right?

Reply:
ah i see, you're right then no need to substitute!

Reply:
done